components/dashboard/src/workspaces/BlogBanners.tsx

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import React, { useEffect, useState } from "react";
+import blogBannerBg from "../images/blog-banner-bg.png";
+
+const banners = [
+    {
+        type: "Blog Post",
+        title: "Gitpod Enterprise: Self-hosted, not self-managed",
+        link: "https://www.gitpod.io/blog/self-hosted-not-self-managed",
+    },
+    {
+        type: "Customer Story",
+        title: "Thousands of hours spent on VM-based development environments reduced to zero using Gitpod",
+        link: "https://www.gitpod.io/customers/kingland",
+    },
+    {
+        type: "Gartner Report",
+        title: `"By 2026, 60% of cloud workloads will be built and deployed using CDE's"`,
+        link: "https://www.gitpod.io/blog/gartner-2023-cde-hypecycle",
+    },
+    {
+        type: "Webinar Series",
+        title: "The Platform Engineering maturity model",
+        link: "https://www.gitpod.io/events#platform-maturity-model-series",
+    },
+];
+
+const initialBannerIndex = 0; // Index for "Self-hosted, not self-managed"
+
+export const BlogBanners: React.FC = () => {
+    const [currentBannerIndex, setCurrentBannerIndex] = useState(initialBannerIndex);
+
+    useEffect(() => {
+        const storedBannerData = localStorage.getItem("blog-banner-data");
```

Comment:
It introduces a new key-value pair in local storage to check the index for blog banners (which would be switched in every 2 days amongst these 4s: 

https://github.com/gitpod-io/gitpod/blob/c9fc1c99a38d2455b4df21ef2c927c1b49d2bdb0/components/dashboard/src/workspaces/BlogBanners.tsx#L10-L31